.github/workflows/health_metrics.yml

```diff
@@ -47,7 +47,8 @@ jobs:
   binary_size_metrics:
     needs: check
```

Comment:
I wonder if we can just remove `needs: check` for `binary_size_metrics` job?

Reply:
No, we need `needs: check` in presubmits. In presubmits, `check` will determine which SDK test to run by updating [these variable](https://github.com/firebase/firebase-ios-sdk/blob/3438778ce70c61f521be29d54e0ffe480d1e6fe2/.github/workflows/health_metrics.yml#L72-L82). 
In postsubmit, we do not need `check` since we will upload all SDK binary data regardless of changing SDKs in PRs.